### ISSUE TITLE
Unifi UNAS

### ISSUE THREAD

**Issue by Jordo-o** [dciancu/unifi-protect-unvr-docker-arm64#14]
I've tried to convert this repository to the new Unifi UNAS Pro.
I arrived at the setup, but won't detect the storage drive(s).

<img width="1073" alt="image" src="https://github.com/user-attachments/assets/e6d06a0e-881e-4be9-8c07-8c912771db2e">

Is there anyone who can help with this issue?
It looks like the new UNAS is detecting storage drives in an other way then the UNVR.

**Comment by RobBie1221** [comment]
See #9 

With this repo based on Unifi OS 4 I haven't found someone yet who is able to capture storage. I tried many things myself, and there are some messages pointing to errors in the logs, but haven't been able to figure out root cause. Going back to Unifi OS 3 worked for me.

**Comment by Jordo-o** [comment]
Yeah, I understand that the storage page doesn't work. But I had protect working with Unifi OS 4, during setup it found the storage. 

But with the UNAS, the setup doesn't work. So I hope that is different, because the new UNAS doesn't work with OS 3.

**Comment by RobBie1221** [comment]
It’s not only the storage page for me, it never detects drives with OS 4 (see my comment on that page)

**Comment by dciancu** [comment]
@Jordo-o I don't see any repo under your username, thus I can't take a look at it, but still it is not something I personally am interested to invest time into. For NAS there are so good open source alternatives (see [OpenMediaVault](https://www.openmediavault.org/) for example and many others exist), imho it is not worth the effort due to the time such a project requires to be maintained. This repo is focused on UNVR for Protect only. I will keep this issue open, in case others would like to join, but publish your repo.

**Comment by JPBeltman** [comment]
Hi! Sorry for commenting on a closed issue but do you still have interest in a Unifi Drive container? 

I have a POC running on my rPi4b, emulating a UNAS2, but it still needs a lot of work and testing; the basic storage things work: smb, nfs, timemachine but haven't looked at the plugins & disaster recovery etc. 

If you're interested I will invite you to my private repo; will eventually make it public but it's a bit too early for that at this stage

**Comment by Jordo-o** [comment]
@JPBeltman thanks for your message! 
For me personally, I'm not interested anymore since I have bought a NAS solution. But it is very nice that you still try to make one! 

I don't have much time to help with the programming, but maybe I can help with some testing if you want. 😁
